app/uk/gov/hmrc/agentsubscription/GuiceModule.scala

```diff
@@ -38,8 +38,8 @@ class GuiceModule(environment: Environment, configuration: Configuration) extend
     bind(classOf[HttpPost]).toInstance(WSHttp)
     bind(classOf[HttpGet]).toInstance(WSHttp)
     bind(classOf[DesConnector])
-    bind(classOf[AuthConnector])
     bind(classOf[AuditConnector]).toInstance(MicroserviceGlobal.auditConnector)
```

Comment:
May we avoid having a global object here and use DI?

Reply:
addressed